app/models/exception_hunter/error.rb

```diff
@@ -26,6 +26,10 @@ class Error < ::ExceptionHunter::ApplicationRecord
       joins(:error_group).where(error_group: ErrorGroup.resolved)
     }
 
+    scope :from_ignored_error_groups, lambda {
```

Comment:
what about a small refactor here?
```
scope :from_error_groups, lambda { |group|
      joins(:error_group).where(error_group: group)
    }
```


@brunvez WDYT?

Reply:
I think that would be more focused onto a particular error group, maybe you want to display all errors disregarding their error group by state 🤔

Reply:
Agree, but we already have `from_active_error_groups`, `from_resolved_error_groups` that share the same logic.

BTW I do not consider this a must, just a suggestion 😄

Reply:
I think this is alright, I think at the end of the day ErrorStatus will be a limited set. No need for that much indirection